.github/workflows/release.yml

```diff
@@ -14,6 +14,12 @@ on:
         required: false
         description: ''
         default: ''
+      pre_release:
+        type: boolean
+        required: true
+        description: ''
+        default: true
+
   workflow_call:
     inputs:
       commit:
```

Comment:
If `workflow_call` is triggered, will `pre_release` default to `false`? Or is it set some other way?

Reply:
I would expect `pre_release` will default to not being set at all in this situation. I don't think the inputs between the different triggers behave with a global scope.

Reply:
Where is `workflow_call` for release.yml called from, should this be removed?